tests/tests.go

```diff
@@ -232,14 +236,14 @@ func DeleteLeaderManager() {
 
 func changeManagerReplicas(numOfReplica int32) error {
```

Comment:
please change the last Eventually's error description from: "failed to change kubemacpool deployment number of replicas" to "failed to change kubemacpool statefulset number of replicas"